Classes/WDResizableCropOverlayView.swift

```diff
@@ -192,31 +204,62 @@ internal class WDResizableCropOverlayView: WDImageCropOverlayView {
         if newFrame.size.width < 64 {
             newFrame.size.width = cropBorderView.frame.size.width
             newFrame.origin.x = cropBorderView.frame.origin.x
+            if lockAspectRatio {
```

Comment:
Equilibrates changes between width & height to keep ratio (just like all other changes made in this function).